plone/recipe/codeanalysis/__init__.py

```diff
@@ -293,6 +293,13 @@ def parse_command_line_arguments(options):
     return options
 
 
+def _taskrunner(klass, options, lock, status):
+    check = klass(options, lock)
+    if check.enabled:
+        if not check.run():
+            status.value = False
+
+
```

Comment:
I don't mind inner methods; but if you do I'm surprised the method gets factored out but the inner class does not.